db-engine/src/main/java/com/splicemachine/db/impl/sql/GenericColumnDescriptor.java

```diff
@@ -230,9 +230,19 @@ public static String readNullableUTF(ObjectInput in) throws IOException {
      * @exception IOException thrown on error
      */
     public void writeExternal(ObjectOutput out) throws IOException {
-        writeNullableUTF(out, name);
-        writeNullableUTF(out, tableName);
-        writeNullableUTF(out, schemaName);
```

Comment:
You can keep the `writeNullableUTF` calls for tableName and schemaName, right?

Reply:
as it's hard to reproduce i'd rather change as little as possible

Reply:
+1 if you can test and validate